app/src/main/java/com/jerboa/ui/components/comment/CommentNode.kt

```diff
@@ -204,6 +206,7 @@ fun LazyListScope.commentNodeItem(
     account: Account?,
     isCollapsedByParent: Boolean,
     showActionBar: (commentId: Int) -> Boolean,
+    siteView: SiteView,
```

Comment:
@twizmwazin I'm leaning towards only passing *just* the `enable_downvotes` setting, rather than the entire `SiteView`, to potentially prevent rerenders. What do you think?

Reply:
Just passing the `enable_downvotes` setting is consistent with the current approach. That said, this is the fifth setting being passed here, so I want to continue exploring alternatives.

Reply:
Thanks. I'll get on it